.github/label-actions.yml

```diff
@@ -0,0 +1,34 @@
+# When `devnet-e2e-test` is added, also assign `devnet` to the PR.
+devnet-e2e-test:
+  prs:
+    label:
+      - devnet
+
+# When `devnet-e2e-test` is removed, also delete `devnet` from the PR.
+-devnet-e2e-test:
+  prs:
+    unlabel:
+      - devnet
+
+# When `devnet` is added, also assign `push-image` to the PR.
+devnet:
+  prs:
+    label:
+      - push-image
+
+# When `devnet` is removed, also delete `push-image` from the PR.
+-devnet:
+  prs:
+    unlabel:
+      - push-image
+      - devnet-e2e-test
```

Comment:
Confirming this is intented?

Reply:
That is a hypothetical scenario I don't think we will encounter often, if ever. After giving some thought, I'll just keep `devnet-e2e-test` in that array.